masq_lib/Cargo.toml

```diff
@@ -12,9 +12,11 @@ actix = "0.7.9"
 clap = "2.33.3"
 const_format = "0.2.22"
 crossbeam-channel = "0.5.1"
+csv = "1.3.0"
 dirs = "4.0.0"
 ethereum-types = "0.9.0"
 itertools = "0.10.1"
+ip_country = { path = "../ip_country"}
```

Comment:
Bert suggests that perhaps `ip_country` shouldn't have to be part of `masq_lib`, but could reference `masq_lib` and be referenced  directly by other subprojects that need it.

masq_lib <- ip_country
masq_lib <- node
ip_country <- node

Reply:
shared_schema needs to validate country code, for that I am using const INDEX_BY_ISO3166 from ip_country